src/sql/workbench/contrib/welcome/page/browser/gettingStartedTour.css

```diff
@@ -0,0 +1,228 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.ads_tour:focus {
+	outline: 0 none !important;
+}
+
+.monaco-workbench>.ads_tour {
```

Comment:
Not sure why you chose to use `_` but everywhere else we use `-` as you can see in `.monaco-workbench`